week07/todolist/src/components/TodoList.js

```diff
@@ -0,0 +1,21 @@
+import TodoItem from "./TodoItem";
+
+function TodoList({ todoList, setTodoList, completedList, setCompletedList }) {
+    return (
+        <ul className="todo-list">
+            {todoList.map((todoItem) => {
+                return (
+                    <TodoItem
+                        todoThis={todoItem}
+                        todoList={todoList}
+                        setTodoList={setTodoList}
+                        completedList={completedList}
+                        setCompletedList={setCompletedList}
+                    />
+                );
+            })}
```

Comment:
P1: map 함수를 사용할 때에는 child 각각에 unique한 key값을 부여해야 합니다. 보통 key는 id와 동일한 값을 사용합니다. 현재 컴파일을 하면, 아마 콘솔에 warning 표시가 나타날거예요! 다음과 같이 수정한다면 경고가 사라지는 것을 확인할 수 있습니다.

```suggestion
            {todoList.map((todoItem) => {
                return (
                    <TodoItem
                        key = {todoItem.id}
                        todoThis={todoItem}
                        todoList={todoList}
                        setTodoList={setTodoList}
                        completedList={completedList}
                        setCompletedList={setCompletedList}
                    />
                );
            })}
            
```